themes/classic/templates/checkout/_partials/steps/shipping.tpl

```diff
@@ -79,6 +79,15 @@
             </div>
           {/block}
           <div class="order-options">
+            {if $separatePackages.allowed}
+              <span class="custom-checkbox">
+              <input class="js-gift-checkbox" id="allow_separated_package" name="allow_separated_package"
```

Comment:
```suggestion
              <input id="allow_separated_package" name="allow_separated_package"
```

In the end, I don't we need it

Reply:
but don't trust me with this one, if there are some visual aspects related to this class, we need to make sure that we're add a dedicated one

Reply:
Honestly, I think it comes from the source code I copied from PrestaShop 1.6.
This PR has almost 1 year now … and some reviewed was made.
If the class is not used, I know it makes some dead code … but It worked as this … Now, I have to check if the removal of the class is ok or not. It will dellay (again) the resolution for just a class attribute …

Reply:
There is some js code which use this class : 
themes/classic/_dev/js/checkout.js
themes/classic/assets/js/theme.js
tests/Selenium/specs/checkout.js

So I don't think we have to remove the class.

Reply:
Or maybe I missused the class …
Is it the class for the gift checkbox maybe ?

Reply:
I think you're right … We should remove this class …